<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelToppic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>190</height>
    </rect>
   </property>
   <property name="text">
    <string>背景</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>280</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>280</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelMin">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>5</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string> -</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelClose">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string> X</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSetting">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>set</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAvator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtLogin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRegister">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelForget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtRegister">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtReturn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxForget">
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxAuto">
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditNickName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>280</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelNickName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditNum</tabstop>
  <tabstop>pbtLogin</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pbtRegister</tabstop>
  <tabstop>pbtReturn</tabstop>
  <tabstop>lineEditNickName</tabstop>
  <tabstop>checkBoxForget</tabstop>
  <tabstop>checkBoxAuto</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
